src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1089,6 +1090,11 @@
                 "enabled": "true"
             }
         },
+        "SAG": {
+            "GLOBAL": {
+                "gwmac": "00:11:22:33:44:55"
```

Comment:
s/gwmac/gateway_mac?

Reply:
Got it, will update it and also change the schema to change gwmac to gateway_mac